src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -628,10 +628,13 @@ public static void Append_CharArray_Invalid()
             Assert.Throws<ArgumentNullException>("value", () => builder.Append((char[])null, 1, 1)); // Value is null, startIndex > 0 and count > 0
 
             Assert.Throws<ArgumentOutOfRangeException>("startIndex", () => builder.Append(new char[0], -1, 0)); // Start index < 0
-            Assert.Throws<ArgumentOutOfRangeException>("count", () => builder.Append(new char[0], 0, -1)); // Count < 0
+            Assert.Throws<ArgumentOutOfRangeException>(
```

Comment:
These are actual bugs that we should have filed so that we can ensure the behavior matches between .NET Core and .NET Framework. We shouldn't blindly condition these, we should be very specific about making this change.  If these are just about the parameter name for example I would much prefer we rename the parameter in .NET Core to match .NET Framework.